notebook.ipynb

```diff
@@ -54,13 +54,13 @@
    "execution_count": null,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    )
</pre><p>I would remove the output from this cell and leave only the image (we have in this case duplicated pipeline images)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-call-center/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>